### PR TITLE
Write lesson explaining the DTO pattern and how to use it

### PR DESCRIPTION
**Pull request by kcleland0818**
This change is being made in the consumer canvas per the conversation on 7/22 to keep the same modules for Blackrock and AWS to streamline.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,242 @@
...
+have our application process the GET request. The only difference here is we'll
+return a DTO instead of a `String`. When we do this, it will serialize the data
+into a JSON object to return.
+
+## Passing in a JSON
+
+Let's run the application and see what happens. We will first need to add a
+football team to the application.
+
+Open up Postman and under the "Headers" tab, add "Content-Type":"application/json"
```
Comment (developerjaym) [inline README.md]: I believe Postman will automatically set the Content-Type to application/json if you choose JSON from the dropdown in the body tab, too.

**Review by developerjaym — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kcleland0818) [reply README.md]: I didn't know that! Good to know!

**Review by linda-seiter — COMMENTED** [review]
Just an FYI, my headers were hidden by default in Postman and I had to click on the link to show them.  You might want to mention that.

**Review by linda-seiter — APPROVED** [review]

**Review by linda-seiter — APPROVED** [review]

**Comment by kcleland0818** [comment]
@linda-seiter and @developerjaym - I added a small section here mentioning the `@JsonIgnore` and `@JsonProperty` annotations, as this may help students with their projects.

Jay, I noticed in a lot of the student submissions regarding the camper lab that students were using the `@JsonIgnore` already. But since I didn't see it formally written in the curriculum, I decided it might be best to add it here.

**Review by developerjaym — APPROVED** [review]

**Review by linda-seiter — APPROVED** [review]

**Pull request merged**